services/auth-service/src/main/resources/application.yml

```
@@ -4,6 +4,6 @@ spring:
   application:
     name: auth-service
   datasource:
-    url: jdbc:mysql://localhost:5433/subscription-tracker
+    url: jdbc:postgresql://localhost:5433/subscription-tracker
     username: ${POSTGRES_USER}
     password: ${POSTGRES_PASSWORD}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)